projects/canopy/src/lib/orientation/docs/orientation.stories.ts

```diff
@@ -0,0 +1,81 @@
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+
+import { LgCardModule } from '../../card';
+import { LgOrientationModule } from '../orientation.module';
+import { LgOrientationDirective } from '../orientation.directive';
+import { LgPaddingModule } from '../../spacing/padding/padding.module';
+
+const responsiveCategory = 'Responsive';
+
+export default {
+  title: 'Helpers/Directives/Orientation/Examples',
+  decorators: [
+    moduleMetadata({
+      imports: [ LgOrientationModule, LgCardModule, LgPaddingModule ],
+    }),
+  ],
+  parameters: {
+    a11y: {
+      disable: true,
+    },
+    backgrounds: {
+      default: 'Super Blue',
+    },
+  },
+  argTypes: {
+    orientation: {
+      name: 'orientation',
+      table: {
+        category: responsiveCategory,
+      },
+    },
+  },
+} as Meta;
+
+const orientationStory: Story<LgOrientationDirective> = (
```

Comment:
The examples with/without orientation seem to behave the same 😅 I think because of the type of content. Or are we missing a `display: flex` somewhere?

Reply:
I forgot to update the ``lg-card-hero-img`` component in those stories 😅 
They should behave differently now, one having `horizontal` orientation on `md` and `lg` breakpoints.